lib/flex_ui/widgets/search_results_header.dart

```diff
@@ -1,54 +1,101 @@
 import 'package:auto_route/auto_route.dart';
+import 'package:flex_storefront/flex_ui/tokens/colors.dart';
+import 'package:flex_storefront/flex_ui/tokens/sizes.dart';
 import 'package:flex_storefront/product_list/cubits/product_search_cubit.dart';
 import 'package:flex_storefront/router.dart';
 import 'package:flex_storefront/search/models/search_results.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_breadcrumb/flutter_breadcrumb.dart';
+import 'package:icons_plus/icons_plus.dart';
 
 class SearchResultsHeader extends StatelessWidget {
   const SearchResultsHeader({
     super.key,
+    this.breadcrumbs = const [],
     required this.searchResults,
   });
 
+  final List<Breadcrumb> breadcrumbs;
   final SearchResults searchResults;
 
   @override
   Widget build(BuildContext context) {
-    return Row(
-      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
       children: [
-        Text('${searchResults.pagination.totalResults} items'),
+        if (breadcrumbs.isNotEmpty)
+          BreadCrumb(
```

Comment:
Hmmm widget named `BreadCrumb`, model named `Breadcrumb`... there should be a better way 😄  (but I don't find any on top of my head)